src/fidesops/graph/config.py

```diff
@@ -233,6 +269,16 @@ def cast(self, value: Any) -> Optional[Any]:
 
         return value
 
+    def getChild(self, names: Tuple[str, ...]) -> Optional["Field"]:
+        """Scalar field is defined as not having children"""
+        return self if len(names) == 0 else None
+
+    def collect_matching(self, f: Callable[[Field], bool]) -> Dict[FieldPath, Field]:
+        """Find fields or subfields satisfying the input function"""
+        if f(self):
+            return {FieldPath(self.name): self}  # pylint: disable=no-member
+        return {}
+
 
 class ObjectField(Field):
```

Comment:
Hmm, ObjectFields can have data_categories and their sub fields can each have data_categories.  How do we handle erasures for data categories where there is a mismatch between the top level and lower levels?

Say we have an ObjectField with data_category A, the has two `fields` one with data_category A and one with data_category B.  If we have a policy to erase all data with category "A" - there is a conflict here -

Reply:
Yes. logically the top level should win, IMO. You can make an argument for more finer grained control but I think that's overkill at the risk of serious overcomplication - even testing and documenting the behavior of that would be complicated and time consuming.

I think best course would be that specifying categories in a subfield is a configuration validation error. Should return a _very_ clear message.

Reply:
thanks for the comment, will save this for the follow-up PR